packages/components/src/avatar/src/Avatar.css

```diff
@@ -82,6 +82,11 @@
     list-style-type: none;
 }
 
+.o-ui-avatar-group-remainings-list-item {
```

Comment:
Since the list shows on hover and I don't think we have a programatic way of enforcing the remaining list to be opened by default I thought it was not worth it?

Reply:
I am not sure if the repo is setupted correctly but this is a perfect use case for a SB play function:

https://storybook.js.org/docs/react/writing-stories/play-function

Reply:
I will test this. Good first dip in the interaction testing.